os/net/lwip/src/api/sockets.c

```diff
@@ -61,7 +61,7 @@
 #include <net/lwip/netif.h>
 #include <netinet/in.h>
 
-#if defined(CONFIG_ARCH_CHIP_S5JT200) || defined(CONFIG_ARCH_CHIP_LM3S6965)
+#if defined(CONFIG_ARCH_CHIP_S5JT200) || defined(CONFIG_ARCH_CHIP_LM3S6965) || defined(CONFIG_ARCH_CHIP_BCM4390X)
```

Comment:
same as previous.
Only this line is valid on this file. Others make violations of coding rule.
Let's rever other changes.

Reply:
Please check this coment as well